curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-functional-programming-by-building-a-spreadsheet/6449842c6f6c84261075e4c9.md

```diff
@@ -37,7 +39,13 @@ Your `idToText` function should have an `id` parameter.
 assert.match(code, /const\s+evalFormula\s*=\s*\(\s*x\s*,\s*cells\s*\)\s*=>\s*{\s*const\s+idToText\s*=\s*(\(\s*id\s*\)|id)\s*=>/);
 ```
 
-You should assign `idToText` the result of calling the `.find()` method on your `cells` array.
+You should assign `idToText` the result of calling the `.find()` method on your `cells` array. 
```

Comment:
```suggestion
Your `idToText` function should return the result of calling the `.find()` method on your `cells` array.
```

This assert message isn't worded correctly (I know you didn't write it).

`idToText` contains a function, it doesn't contain the function's own return value.